R/splatPop-simulate.R

```diff
@@ -459,17 +459,15 @@ splatPopParseVCF <- function(vcf, params){
 #' @param params splatPopParams object containing parameters for population
 #'        scale simulations. See \code{\link{splatPopParams}} for details. 
 #'        Default=`newSplatPopParams()`.
-#' @param gff Either FALSE (if random genes desired) or data.frame object 
-#'        of a GFF/GTF file.
+#' @param gff Either NA or a data.frame object containing a GFF/GTF file.
 #'
 #' @return The partial eQTL key dataframe.
 #'
 splatPopParseGenes <- function(params, gff){
     
-    random.genes <- getParam(params, "random.genes")
     nGenes <- getParam(params, "nGenes")
     
-    if(random.genes){
+    if(is.logical(gff)){
```

Comment:
You probably want `is.na()` here (or `is.null()`)

Reply:
I've removed random.genes from the splatPopParam object and have just made the function determine default to random if genes aren't provided in a key or a gff.